# Xernix rebrand of CRYPTONOTE_NAME
#
# Changes the on-disk data dir and binary identity from "bitmonero" to
# "xernix". This single #define controls:
#   * Default --data-dir on Linux/macOS -> ~/.xernix/
#   * Wallet file naming prefix
#   * Various internal log strings
#
# Pinned against MONERO_REF=v0.18.3.4.
--- a/src/cryptonote_config.h
+++ b/src/cryptonote_config.h
@@ -158,7 +158,7 @@
 
 #define RPC_IP_FAILS_BEFORE_BLOCK                       3
 
-#define CRYPTONOTE_NAME                         "bitmonero"
+#define CRYPTONOTE_NAME                         "xernix"
 #define CRYPTONOTE_BLOCKCHAINDATA_FILENAME      "data.mdb"
 #define CRYPTONOTE_BLOCKCHAINDATA_LOCK_FILENAME "lock.mdb"
 #define P2P_NET_DATA_FILENAME                   "p2pstate.bin"
